src/amber/cli/templates/app/.amber.yml.ecr

```diff
@@ -2,3 +2,27 @@ type: app
 database: <%= @database %>
 language: <%= @language %>
 model: <%= @model %>
+watch:
+  server: # required: the first command for this task is blocking
```

Comment:
What does this mean `required: the first command for this task is blockin`?

Reply:
That means that other tasks are blocked until your amber app compiles

Maybe we should change the message, WDYT? :sweat_smile: